### DISCUSSION TITLE
ortur laser switches on randomly when border tracing

### DISCUSSION THREAD

**Discussion by n10cfi** [arkypita/LaserGRBL#1177] [Q&A]
Hi Everyone,

I have an ortur laser master and i have had a problem appear yesterday. When doing a boarder trace the laser will engage randomly and seemingly without input from laserGRBL which is damaging my work pieces. Have looked at power setting etc but it looks like the laser is just deciding to burn at will.

Any ideas/help greatly appreciated

**Comment by arkypita** [comment]
Fisrt: check cables and connectors.
Second: go to ortur user group or ortur helpdesk and ask there (you find link in ortur menu in LaserGRBL)
